lib/mobility-core/src/Kernel/External/Payment/Juspay/Flow.hs

```diff
@@ -87,3 +112,81 @@ getCurrentDate = do
   let dateFormat = "%Y-%m-%d"
       formattedDate = formatTime defaultTimeLocale dateFormat currentTime
   return $ encodeToText formattedDate
+
+type MandateNotificationAPI =
+  "mandates"
+    :> Capture "merchantId" Text
+    :> BasicAuth "username-password" BasicAuthData
```

Comment:
Can you tell please which is this api? Can't find it in Juspay docs
I was able to find only `NOTIFICATION_SUCCEEDED` event in webhooks https://docs.juspay.in/hyper-checkout/ios/base-sdk-integration/webhooks

Reply:
Please refer here for mandate notification api :- https://developer.juspay.in/reference/notification-api